ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -66,6 +66,7 @@ type ContainerDef struct {
 	MemoryReservation libYaml.MemStringorInt `yaml:"mem_reservation"`
 	HealthCheck       *HealthCheck           `yaml:"healthcheck"`
 	Secrets           []Secret               `yaml:"secrets"`
+	Gpu               string                 `yaml:"gpu"`
```

Comment:
I realize that in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ResourceRequirement.html) this field is specified as a string, but can we verify if there's any good reason why this can't be a number? we can always convert to string in our own code (though of course there is the annoying issue of the default empty value of an int being 0 in go, so would have to make sure that's handled correctly).

Reply:
I think it's best to make it a string since that's what the API takes, and for the default value reason which you mentioned.

Reply:
Also nit: This should be "GPU" to fit golang initialism practices

Reply:
@SoManyHs I went with string to keep it consistent with the API as @PettitWesley pointed it out. I'm scared of putting a different type than the backend in case they enable for example values like `"two"`. Then our solution would be more limiting.
@PettitWesley fixed the variable name. I wrote that because I saw `Cpu` a few lines above it 😜